packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -626,6 +626,15 @@ export class DataStores implements IDisposable {
         }
         return outboundRoutes;
     }
+
+    /**
+     * Returns the package path of the data store with the given id, if the data store exists.
+     */
+    public getDataStorePackagePath(id: string): readonly string[] | undefined {
+        const internalId = this.aliasMap.get(id) ?? id;
```

Comment:
I do not like this leakage of alias API. I think one of the following should happen:
- we do not use IDs at all, we pass objects around
- We use only one type of ID everywhere (internal ID), no usage of alias ID - it's for higher level constructs / user interface only
- We abstract it away somewhere such that every function does not need to know about presence and need to check for aliases.

I think # 2 is the most useful suggestion for this code.

Reply:
I will talk to Andrei and confirm.

Reply:
@andre4i FYI